packages/@react-spectrum/combobox/stories/ComboBox.stories.tsx

```diff
@@ -223,23 +245,149 @@ let customFilterItems = [
   {name: 'The third item', id: '3'}
 ];
 
-let CustomFilterComboBox = () => {
+let CustomFilterComboBox = (props) => {
   let [list, setList] = React.useState(customFilterItems);
+  let [selectedKey, setSelectedKey] = React.useState(props.selectedKey);
+
+  let onSelectionChange = (key) => {
+    setSelectedKey(key);
+  };
 
   let onFilter = (value) => {
     setList(customFilterItems.filter(item => item.name.includes(value)));
   };
 
   return (
-    <ComboBox items={list} itemKey="id" label="Combobox" onFilter={onFilter} onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onSelectionChange={action('onSelectionChange')} onBlur={action('onBlur')} onFocus={action('onFocus')}>
+    <ComboBox items={list} itemKey="id" selectedKey={selectedKey} label="Combobox" onSelectionChange={onSelectionChange} onFilter={onFilter} onOpenChange={action('onOpenChange')} onInputChange={action('onInputChange')} onBlur={action('onBlur')} onFocus={action('onFocus')} onCustomValue={action('onCustomValue')}>
       {(item: any) => <Item>{item.name}</Item>}
     </ComboBox>
   );
 };
 
+let AllControlledComboBox = (props) => {
```

Comment:
Not 100% sure what the expected behavior should be here if the user changes only props.selectedKey or props.inputValue, should discuss